Mathlib/Analysis/Complex/Polynomial/Basic.lean

```diff
@@ -28,8 +28,8 @@ open scoped ComplexConjugate
 
 namespace Complex
 
-/-- **Fundamental theorem of algebra**: every non constant complex polynomial
-  has a root -/
+/-- **Fundamental theorem of algebra**: every nonconstant complex polynomial
```

Comment:
hyphen, please (or, as for all these comments, match local style)

Reply:
See general comment below.

Also, in this file there is actually another case of "nonconstant" being used. But there is also usages of "non-real". Not sure what weighs more.